.github/workflows/test-mobile-e2e.yml

```diff
@@ -55,5 +55,5 @@ permissions:
 jobs:
   test-mobile-e2e:
     name: "Test Mobile E2E"
-    uses: LedgerHQ/ledger-live/.github/workflows/test-mobile-e2e-reusable.yml@develop
+    uses: LedgerHQ/ledger-live/.github/workflows/test-mobile-e2e-reusable.yml@support/qaa_312_detox_speculos_add_accounts
```

Comment:
```suggestion
    uses: LedgerHQ/ledger-live/.github/workflows/test-mobile-e2e-reusable.yml@develop
```

Reply:
I am just waiting for the checks to pass before reverting last commit